components/organisms/cards/statement/statement.vue

```diff
@@ -0,0 +1,236 @@
+                        <download-icon class="mr-2" />
+                        <p class="d-none d-sm-inline m-0">download</p>
+                    </b-button>
+                </div>
+                <div v-if="isCardActive" class="ml-2 ml-sm-5">
+                    <b-button
+                        class="px-0 d-flex align-items-center font-weight-lighter text-decoration-none no-focus"
+                        variant="link"
+                        @click="confirmDeleteCard"
+                    >
+                        <delete-icon class="mr-2" />
+                        <p class="d-none d-sm-inline m-0">delete card</p>
+                    </b-button>
+                </div>
+            </b-col>
+        </b-row>
+        <b-row v-if="filteredStatement">
+            <b-col>
+                <b-row v-for="(statementEntries, date) in filteredStatement" :key="date">
+                    <b-col>
+                        <b-row class="mt-4">
+                            <b-col class="text-muted">
+                                {{ formatDate(date) }}
+                            </b-col>
+                        </b-row>
+                        <b-row v-for="(statement, key) in statementEntries" :key="key">
+                            <b-col>
+                                <statement-item :transaction="statement" />
+                            </b-col>
+                        </b-row>
+                    </b-col>
+                </b-row>
+            </b-col>
+        </b-row>
+    </div>
+</template>
+<script lang="ts">
+import {
+    defineComponent,
+    computed,
+    PropType,
+    useRoute,
+    useRouter,
+    getCurrentInstance,
+} from '@nuxtjs/composition-api'
+import dot from 'dot-object'
+import { AxiosError } from 'axios'
+import { DateTime } from 'luxon'
+import { useStores } from '~/composables/useStores'
+import { useCards } from '~/composables/useCards'
+import { useBase } from '~/composables/useBase'
+import { useFilters } from '~/composables/useFilters'
+import { useRouterFilter } from '~/composables/useRouterFilter'
+import { GetManagedCardStatementRequest } from '~/plugins/weavr-multi/api/models/managed-instruments/managed-cards/requests/GetManagedCardStatementRequest'
+import { OrderEnum } from '~/plugins/weavr-multi/api/models/common/enums/OrderEnum'
+import { ManagedInstrumentStateEnum } from '~/plugins/weavr-multi/api/models/managed-instruments/enums/ManagedInstrumentStateEnum'
+import { CreateTransferRequest } from '~/plugins/weavr-multi/api/models/transfers/requests/CreateTransferRequest'
+import { InstrumentEnum } from '~/plugins/weavr-multi/api/models/common/enums/InstrumentEnum'
+
+export default defineComponent({
+    name: 'CardStatementComponent',
+    components: {
+        DownloadIcon: () => import('~/assets/svg/download.svg?inline'),
+        DeleteIcon: () => import('~/assets/svg/delete.svg?inline'),
+        StatementItem: () => import('~/components/organisms/StatementItem.vue'),
```

Comment:
Can these imports be with the rest?

Reply:
I think this will fail. maybe when we use vite.